shared/desktop/renderer/index.html.template

```diff
@@ -7,7 +7,7 @@
     object-src 'self';
     font-src 'self' <%= htmlWebpackPlugin.options.isDev ? 'http://localhost:4000' : '' %>;
     media-src http://127.0.0.1:*;
-    img-src 'self' data: http://127.0.0.1:* https://keybase.io/ https://pbs.twimg.com/ https://avatars.githubusercontent.com/ https://s3.amazonaws.com/keybase_processed_uploads/ <%= htmlWebpackPlugin.options.isDev ? 'http://localhost:4000' : '' %>;
+    img-src 'self' data: http://127.0.0.1:* http://localhost:* https://keybase.io/ https://pbs.twimg.com/ https://avatars.githubusercontent.com/ https://s3.amazonaws.com/keybase_processed_uploads/ <%= htmlWebpackPlugin.options.isDev ? 'http://localhost:4000' : '' %>;
```

Comment:
Drop this change. @chrisnojima points out changing the CSP could be risky so we can https://github.com/keybase/client/pull/17106 instead

Reply:
oops, forgot about it

Reply:
will remove after remaining issues are sorted out so that I can still test